packages/ui-react/src/components/Sidebar/Sidebar.module.scss

```diff
@@ -29,6 +29,7 @@
   display: none;
   width: 270px;
   max-width: 90vw;
+  height: 100vh; // Fallback
```

Comment:
Maybe this way?

```
.sidebar{
    height: 100vh;
}

@supports (height: 100dvh) {
  .sidebar{
    height: 100dvh;
  }   
}
```

Reply:
The support of this syntax is limited. https://caniuse.com/mdn-css_at-rules_supports_selector

If we going to extend the browser support further, I suggest to let `postcss-preset-env` to take care of the fallbacks. I experimented with this but this `dvh` unit needed a specific plugin and didn't get transformed automatically, unfortunately. 

Since this is the only CSS browser compatibility issue, I vouch for keeping it simple for now without adding a lot of overhead.